fleet-management-spring-mvc/pom.xml

```diff
@@ -19,11 +19,6 @@
             <artifactId>fleet-management-sample-data</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
-        <dependency>
-            <groupId>cz.fi.muni.pa165</groupId>
-            <artifactId>fleet-management-rest</artifactId>
-            <version>${project.parent.version}</version>
```

Comment:
We asked him in exam ... In the example they don't do what we have to, all function for the 3th milestone should run when you start the application including rest.